setup/gcp/installer/configs/server_config.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, **kwargs):
         self.cloudsql_instance = '{}-{}'.format(
             'forseti-security',
             self.datetimestamp)
-        self.cloudsql_region = kwargs.get('cloudsql_region', 'us-central1')
+        self.cloudsql_region = kwargs.get('cloudsql_region') or 'us-central1'
```

Comment:
As this discussed with joe, this would be better for reference to be centralized as flag defaults, rather than implemented in the codebase.